core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/RegistryResources.java

```diff
@@ -160,6 +160,20 @@ public static final class SecurityManagement {
         public static final String SERVER_INTERNAL_PRIVATE_KEY_PASSWORD = "Security.InternalKeyStore.KeyPassword";
         public static final String SERVER_INTERNAL_KEYSTORE_TYPE = "Security.InternalKeyStore.Type";
 
+        //Custom KeyStore related constants
```

Comment:
```suggestion
        // Custom KeyStore related constants.
```

Reply:
Fixed in https://github.com/wso2/carbon-kernel/pull/4025/commits/a0533c76776fd427f270477373ea6951ce2f147f